docs/pages/client-side-routing.mdx

```diff
@@ -0,0 +1,100 @@
+<Meta title="Guides/Client Side Routing" />
```

Comment:
Think you might be missing some of the imports at the top of the document on this page 
![Screenshot 2025-02-06 at 2 57 58 pm](https://github.com/user-attachments/assets/75542fe8-f40a-46f6-a76c-f9724271fee1)